tests/rgw/test_s3.py

```diff
@@ -65,16 +66,16 @@ def setup_s3_tests(client_node, rgw_node, config, build):
     repo_url = "https://github.com/ceph/s3-tests.git"
     client_node.exec_command(cmd="git clone -b {branch} {repo_url}".format(branch=branch, repo_url=repo_url))
 
-    log.info("Running bootstrap")
     if build.startswith('4'):
         client_node.exec_command(cmd="cd s3-tests")
-        client_node.exec_command(cmd="sudo yum install -y python2-virtualenv python2-devel libevent-devel"
-                                 "libffi-devel libxml2-devel libxslt-devel zlib-devel", check_ec=False)
-        client_node.exec_command(cmd="virtualenv --no-site-packages --distribute virtualenv")
-        client_node.exec_command(cmd="./virtualenv/bin/pip install setuptools==32.3.1")
-        client_node.exec_command(cmd="./virtualenv/bin/pip install -r requirements.txt")
-        client_node.exec_command(cmd="./virtualenv/bin/python setup.py develop")
+        client_node.exec_command(cmd="sudo yum install -y python2-virtualenv python2-devel libevent-devel libffi-devel "
```

Comment:
Suggesting a list to maintain installation packages.

Reply:
done.